CrowdStrike/ember-toucan-core
# toucan-form: Update docs, add tests

Downstream of https://github.com/CrowdStrike/ember-toucan-core/pull/137

## 🚀 Description

- Remove headless-form docs
- Update test values entered in tests (per feedback from @nicolechung )
  - https://github.com/CrowdStrike/ember-toucan-core/pull/134#discussion_r1170195156
- Update toucan-form docs
- Add an `onSubmit` verification to the toucan-form test
- Removed markup in `application.hbs` template for `test-app`

---

## 🔬 How to Test

- View pages at https://26dea2f1.ember-toucan-core.pages.dev/docs/toucan-form/changeset-validation to see the visual changes
- Verify no more headless-form examples in the nav
- Green build

---

## 📸 Images/Videos of Functionality

N/A
